src/pages/cli/project/troubleshooting.mdx

```diff
@@ -0,0 +1,263 @@
+On deployment of this application, the Amplify generated CloudFormation template provisions AppSync API models, resolvers, multiple DynamoDB tables and GSIs. 
+
+#### Scenario 1: "amplify push" fails after manually updating DynamoDB tables and GSIs from the console.
+Assume you have an application with a GraphQL schema deployed to the cloud. 
+Amplify creates DynamoDB tables for all [@model]((/cli/graphql/data-modeling/)) types and GSIs for all [@index]((/cli/graphql/data-modeling/)) fields in the GraphQL schema. 
+"Drift" is introduced if you delete any of the GSIs using the DynamoDB console, instead of using the Amplify CLI.
+With drift, future changes to the GraphQL schema may fail to deploy. 
+As an example, if you change the names of some `@model` types and `@index` fields in your GraphQL schema and perform the `amplify push —allow-destructive-graphql-schema-updates` command, Amplify will first remove all DynamoDB tables corresponding to the original model names. 
+However if any of the GSIs on these tables were already deleted manually from the console then the deployment will fail. This is because Amplify’s CloudFormation stack is attempting to update the state of a resource (the GSI) which doesn’t exist. 
+
+
+**Suggested Resolution:**
+To fix deployment failures due to drift in DynamoDB tables, manually rollback the state of the drifted resource to match its state in CloudFormation. Then retry `amplify push`. 
+1. You can view the drift in the DynamoDB CloudFormation Stack per instructions in the [AWS drift detection documentation](/AWSCloudFormation/latest/UserGuide/detect-drift-stack.html)
+1. Revert every drift manually through the AWS console. In this particular scenario, re-create the GSIs and manually undo any renaming changes.
+1. Perform an `amplify push`.
+
+#### Scenario 2: "amplify push" fails after unmanaged updates to non-database resources in the cloud.
+
+Assume you have an application with a GraphQL schema deployed to the cloud using Amplify CLI. 
+You then have performed one or more of the following types of changes  
+1. Updated your GraphQL schema using the AppSync console, or
+2. Deleted user pools created by Amplify from the Cognito console, or
+3. Updated the code in Lambda functions which were originally deployed through Amplify.
+These types of unmanaged changes will introduce a drift for the affected resources. Therefore subsequent updates to these resources using Amplify will most likely fail deployment.
+
+**Suggested Resolution:** 
+These types of drifts must be addressed on a case-by-case basis. In some cases, such as a Lambda code change, forcing a redeployment of the function by making a minor change to the code will likely work. 
+In other cases, like deleting user-pools or updates to GraphQL schema from the AppSync console, follow the steps below:
+1. Forcible deployment of CloudFormation stack from the CLI - Run `amplify push --force` to force push the application’s resource stack to be deployed.
```

Comment:
Can this be simplified to? Not sure if it's semantically/technically accurate to say this
```suggestion
1. Force a new CloudFormation stack deployment - Run `amplify push --force` to force push the application's resources.
```